src/atomate2/vasp/jobs/lobster.py

```diff
@@ -0,0 +1,301 @@
+from pathlib import Path
+
+from jobflow import Flow, Response, job
+from pymatgen.core import Structure
+from pymatgen.io.lobster import Lobsterin
+
+from atomate2.common.files import delete_files
+from atomate2.lobster.jobs import PureLobsterMaker
+from atomate2.lobster.schemas import LobsterTaskDocument
+from atomate2.utils.path import strip_hostname
+from atomate2.vasp.jobs.base import BaseVaspMaker
+from atomate2.vasp.powerups import update_user_incar_settings
+from atomate2.vasp.sets.base import VaspInputGenerator
+from atomate2.vasp.sets.core import StaticSetGenerator
+
+__all__ = ["VaspLobsterMaker", "get_basis_infos", "get_lobster_jobs"]
+
+logger = logging.getLogger(__name__)
+
+
+# include a class where we can also add Lobster
+
+
+@dataclass
+class VaspLobsterMaker(BaseVaspMaker):
+    """
+    Maker that performs a VASP computation with
+     settings that are required for Lobter runs
+    Parameters
+    ----------
+    name : str
+        The job name.
+    input_set_generator : .VaspInputGenerator
+        A generator used to make the input set.
+    write_input_set_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.write_vasp_input_set`.
+    copy_vasp_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.copy_vasp_outputs`.
+    run_vasp_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.run_vasp`.
+    task_document_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.TaskDocument.from_directory`.
+    stop_children_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.should_stop_children`.
+    write_additional_data : dict
+        Additional data to write to the current directory. Given as a dict of
+        {filename: data}. Note that if using FireWorks, dictionary keys cannot contain
+        the "." character which is typically used to denote file extensions. To avoid
+        this, use the ":" character, which will automatically be converted to ".". E.g.
+        ``{"my_file:txt": "contents of the file"}``.
+    """
+
+    name: str = "static_run"
+    input_set_generator: VaspInputGenerator = field(
+        default_factory=lambda: StaticSetGenerator(
+            user_kpoints_settings={"grid_density": 6000},
+            user_incar_settings={
+                "IBRION": 2,
+                "ISIF": 2,
+                "ENCUT": 700,
+                "EDIFF": 1e-7,
+                "LAECHG": False,
+                "LREAL": False,
+                "ALGO": "Normal",
+                "NSW": 99,
+                "LCHARG": False,
+                "LWAVE": True,
+                "ISYM": 0,
+            },
+        )
+    )
+    copy_vasp_kwargs: dict = field(
+        default_factory=lambda: {"additional_vasp_files": ["WAVECAR"]}
+    )
+
+
+@job
+def get_basis_infos(
+    structure: Structure,
+    vaspmaker: BaseVaspMaker,
+    address_max_basis: str = None,
+    address_min_basis: str = None,
+):
+    """
+
```

Comment:
Can you use numpy style docstrings here and add an initial line saying what the function does.

Reply:
Yes, of course. I am so used to google that I forgot that atomate2 is using numpy.